statistics.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"github.com/Iwark/spreadsheet"
+	"regexp"
+	"strings"
+	"time"
+)
+
+type EmployeeAnniversary struct {
+	Name string
+	Date string
+	WorksYears int
+	Color string
+}
+
+func getWorkAnniversaries() [12][]EmployeeAnniversary {
+	ctx := context.Background()
+
+	b, err := ioutil.ReadFile(config.CalendarSecret)
+	if err != nil {
+		log.Fatalf("Unable to read client secret file: %v", err)
+	}
+	calendarConfig, err := google.ConfigFromJSON(b, spreadsheet.Scope)
+	if err != nil {
+		log.Fatalf("Unable to parse client secret file to config: %v", err)
+	}
+	client := getSheetClient(ctx, calendarConfig)
+	service := &spreadsheet.Service{Client: client}
+
+	sheets, _ := service.Get(config.AnniversariesSheetId)
+
+	ws1, _ := sheets.Get(0)
+
+	r, _ := regexp.Compile("([0-9]{4}-[0-9]{2}-[0-9]{2})")
+	rName, _ := regexp.Compile(`[0-9\(\)\-]+`)
+
+	employees := [12][]EmployeeAnniversary{}
+	for i, row := range ws1.Rows {
+		if i == 0 {
+			continue
+		}
+
+		for j := 0; j < 12; j++ {
+			if row[j] != nil {
+				employeeWorkYear, _ := time.Parse("2006-01-02", r.FindString(row[j].Content))
+				worksYears := time.Now().Year() - employeeWorkYear.Year()
+				employee := EmployeeAnniversary{Color: randomColor(),Date: r.FindString(row[j].Content), Name: strings.TrimSpace(rName.ReplaceAllString(row[j].Content, "")), WorksYears: worksYears}
```

Comment:
`row[j].Content` is used more than once, I think it should be extracted to separate variable

Reply:
I'm pretty sure this exceeds 120 symbols per line